packages/common/src/tests.ts

```diff
@@ -0,0 +1,20 @@
+import { InvitationData } from '@quiet/types'
+import { composeInvitationDeepUrl, composeInvitationShareUrl } from './invitationCode'
+import { QUIET_JOIN_PAGE } from './static'
+
+const validInvitationCodeTestData: InvitationData = {
+  pairs: [
+    {
+      onionAddress: 'y7yczmugl2tekami7sbdz5pfaemvx7bahwthrdvcbzw5vex2crsr26qd',
+      peerId: 'QmZoiJNAvCffeEHBjk766nLuKVdkxkAT7wfFJDPPLsbKSE',
+    },
+  ],
+  psk: 'BNlxfE2WBF7LrlpIX0CvECN5o1oZtA16PkAb7GYiwYw=',
+}
+
+export const validInvitationUrlTestData = {
+  shareUrl: () => composeInvitationShareUrl(validInvitationCodeTestData),
+  deepUrl: () => composeInvitationDeepUrl(validInvitationCodeTestData),
+  code: () => composeInvitationShareUrl(validInvitationCodeTestData).split(QUIET_JOIN_PAGE + '#')[1],
+  data: validInvitationCodeTestData,
+}
```

Comment:
I really don't like duplicating code when I can avoid duplicating it. I understand that 'common' package (which should be renamed btw.) may not be the perfect place for test data but I'd leave that here for now. If something changes it's easier to edit one place that few.

Reply:
This is also a nitpick in some ways so I don't think we should spend too much time on it. But it could be worth discussing a little bit more, so I can share some thoughts.

I don't think it's always easier to change something if it's in one place. I think it depends on what code is using that thing. If many different pieces of code are using a single thing, then changing that single thing can involve changing lots of other things. So by referencing a single thing in multiple places, there are now dependencies on that thing.

Only defining everything once can also make the code more difficult to read. Instead of reading a single file, you might have to jump around to many, many files. Everything may only be defined once, but figuring out what's going on now involves tying together a much larger web of files and dependencies.

And so I think the concept of DRY (https://en.wikipedia.org/wiki/Don't_repeat_yourself), when done without considering the trade-offs, is too rigid. In this case, I'm not sure if the de-duplication is worth it. But I also don't think this is that important right now, so I'll follow your intuition.

Reply:
Yes, I agree that this depends on the usage and context. However jumping from file to file is not the best argument since vsc and other code editors make it easy. 
Here we use the same format of invitation code across all project, that's why I decided to put it in a common place, which (I repeat) may not be ideal for test utils but that's what we have right now.
Let's leave it as it is.

Reply:
Sounds good to me, thanks for considering it.